src/index.ts

```diff
@@ -265,7 +265,7 @@ export class SDK {
         this.page = new PageController(this.editorAPI);
         this.debug = new DebugController(this.editorAPI);
         this.undoManager = new UndoManagerController(this.editorAPI);
-        this.textSelection = new TextStyleController(this.editorAPI);
+        this.textStyle = new TextStyleController(this.editorAPI);
```

Comment:
There was a typo: `textSelection` > `textStyle`.

I'd like to limit the amount of breaking-changes so I put it in this PR but this is actually done in scope of https://support.chili-publish.com/browse/EDT-926 which needed this controller name.